io.catenax.week_based_capacity_group/3.0.0/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,424 @@
+   samm:characteristic :Timestamp ;
+   samm:exampleValue "2023-03-10T12:27:11.320Z"^^xsd:dateTimeStamp .
+
+:linkedCapacityGroups a samm:Property ;
+   samm:preferredName "Linked Capacity Groups"@en ;
+   samm:description "A set of capacity groups linked to this capacity group."@en ;
+   samm:characteristic :LinkedCapacityGroupsCharacteristic ;
+   samm:exampleValue "be4d8470-2de6-43d2-b5f8-2e5d3eebf3fd" .
+
+:unitOfMeasureIsOmitted a samm:Property ;
+   samm:preferredName "Unit Of Measure Is Omitted"@en ;
+   samm:description "Explicit indicator of whether the unit of measure is left out of the payload intentionally. If \"true\" it means the sending application sends demand values without unit of measure intentionally and the unit of measure MUST NOT be contained in the payload. If \"false\" a unit of measure MUST be supplied."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
+
+:demandVolatilityParameters a samm:Property ;
+   samm:preferredName "Demand Volatility Parameters"@en ;
+   samm:description "The parameters to calibrate the measuring of demand volatility"@en ;
+   samm:characteristic :DemandVolatilityCharacteristic .
+
+:isInactive a samm:Property ;
+   samm:preferredName "Is Inactive"@en ;
+   samm:description "Indicates that this capacity group is currently not in use by the supplier anymore."@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue true .
+
+:CapacityGroupName a samm:Characteristic ;
+   samm:preferredName "Capacity Group Name"@en ;
+   samm:description "The name of a capacity group, used to distinguish it from other capacity groups."@en ;
+   samm:dataType xsd:string .
+
+:SupplierLocationsCharacteristic a samm-c:Set ;
+   samm:preferredName "Supplier Locations Characteristic"@en ;
+   samm:description "Contains a set of locations the supplier fulfills a demand request from."@en ;
+   samm:dataType xsd:string ;
+   samm-c:elementCharacteristic :BPNSTrait .
+
+:LinkedDemandSeriesSet a samm-c:Set ;
+   samm:preferredName "Linked Demand Series Set"@en ;
+   samm:description "A set of demand series assigned to a certain capacity group."@en ;
+   samm:dataType :LinkedDemandSeries .
+
+:CapacityTimeSeries a samm-c:SortedSet ;
+   samm:preferredName "Capacity Time Series"@en ;
+   samm:description "The weekly actual and maximum capacities in a given time period."@en ;
+   samm:dataType :Capacity .
+
+:Timestamp a samm:Characteristic ;
+   samm:preferredName "Timestamp"@en ;
+   samm:description "Point in time."@en ;
+   samm:dataType xsd:dateTimeStamp .
+
+:LinkedCapacityGroupsCharacteristic a samm-c:Set ;
+   samm:preferredName "Linked Capacity Groups Characteristic"@en ;
+   samm:description "Contains all capacity groups which are linked to the current one."@en ;
+   samm:dataType xsd:string ;
+   samm-c:elementCharacteristic :UUIDv4IdTrait .
```

Comment:
same here, you can directly link here already to the shared aspect

Reply:
@catroest my problem with these two characteristics is that sets don't seem to be allowed to be linked to a trait as an external reference - in the editor i'm getting an error for that. If you can help me with this, I would contact you - maybe it's an operating error for me.

Reply:
normally you can just drag and drop the trait from the workspace and connect it with the property. As it is already there, use the connect button. Did you maybe try to connect it on the characteristic?
![image](https://github.com/eclipse-tractusx/sldt-semantic-models/assets/92307971/87d0aa62-8306-44aa-8ec4-f526ea234a7c)

Reply:
I see the problem as well. In the normal structure a trait is only done "above" a chracteristic. So i am not really sure that something like this is intended. Because if you disconnect your own trait with the set, it isn't allowed to reconnect.

Maybe recheck with someone from the SAMM-Tool to ensure this connection is "ok" to use and does not lead to problems. If there are no problems, I would stay as it is ...

[image](https://github.com/eclipse-tractusx/sldt-semantic-models/assets/92307971/595aa858-9de8-4760-85f9-dfbe3b314123)

Reply:
@agg3fe can you please take a quick look at the problem described above and give us some feedback on why the tool behaves this way in this case? If the modeling is not intended in such a way that sets can be connected directly to a trait, then I would like to make a short appointment with you to understand the correct modeling at this point and to implement it correctly. Thanks in advance!

Reply:
let me have a look into it.